app/src/main/java/com/asuc/asucmobile/fragments/MapsFragment.java

```diff
@@ -335,8 +339,8 @@ public void onClick(View v) {
                 bundle.putString("Category", "microwaves");
                 mFirebaseAnalytics.logEvent("map_icon_clicked", bundle);
 
-                if (showSpotlight && viewedSpotlight()) {
-                    mFirebaseAnalytics.logEvent("map_icon_clicked_after_spotlight", bundle);
+                if (viewedFirstSession()) {
+                    mFirebaseAnalytics.logEvent("map_icon_clicked_after_first_session", bundle);
                 }
 
```

Comment:
I don't see something that registers a click on a microwave icon? Shouldn't there be one similar to nappods and water fountains? If so, the Category should be "Microwave" to match iOS styling so Firebase registers them as the same event.